pages/_app.js

```diff
@@ -1,10 +1,18 @@
+import { useState } from "react";
 import GlobalStyle from "../styles";
 
 export default function App({ Component, pageProps }) {
-	return (
-		<>
-			<GlobalStyle />
-			<Component {...pageProps} />
-		</>
-	);
+  //Hier States
```

Comment:
You might consider to leave out comments in general if they do not fulfill an additional value for the reader. In this case I would not understand what is meant by it. You probably put this comment while coding to remind you of implementing a state here.